context-free.md

```diff
@@ -1,22 +1,37 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for people who want to create digital art.
 
 ## What is easy to do in this language? Why is it easy?
-
+Recursively creating shapes is easy because shapes are created inside rules.
```

Comment:
I definitely agree.  Along with recursion it also seems that one of the strengths is the ability to have grammars which could be expanded probabilistically to draw L-systems and the like, by simply placing a above rules to determine which one should replace the original shape.  Unfortunately, the documentation of this section was a bit lacking, so although it was easy to _use_ if its presence is known, it does not seem that the feature was easy to discover. I know I didn't find it until I had already decided to make a deterministic drawing.